packages/components/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.7.0 (Unreleased)
```

Comment:
> This is a breaking change, but we are still on the first version so I _think_ its ok not to bump to `2.0.0`. I'm open to bumping though.

I think we actually do want to release a `2.0.0`. The npm modules are using semver and we've previously published the other versions. We have also already done a `2.0.0` on the navigation package: https://github.com/woocommerce/wc-admin/blob/master/packages/navigation/CHANGELOG.md.